conf/machine/amlogic-s9xxx.conf

```diff
@@ -32,5 +32,6 @@ require conf/machine/include/oranth-tx3-mini-dtb.inc
 require conf/machine/include/amlogic-u200-dtb.inc
 require conf/machine/include/hardkernel-odroidn2-dtb.inc
 require conf/machine/include/khadas-vim3-dtb.inc
-require conf/machine/include/khadas-vim3-s922x-dtb.inc
```

Comment:
Why did you drop the s922x dtb ?

Reply:
> Why did you drop the s922x dtb ?

See the first patch in the series.  It's included in the vim3-dtb.inc.